util/exec_test.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/symflower/eval-dev-quality/log"
+)
+
+func TestCommandWithResult(t *testing.T) {
+	logOutput, logger := log.Buffer()
+	defer func() {
+		if t.Failed() {
+			t.Log(logOutput.String())
+		}
+	}()
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(1*time.Second))
+	defer cancel()
+
+	start := time.Now()
+	CommandWithResult(ctx, logger, &Command{
+		Command: []string{
+			"sleep",
+			"60",
+		},
+	})
```

Comment:
Please assert that there is an error returned so we know that hitting the timeout produces an error.